### DISCUSSION TITLE
I have some problems upgrading to Wagtail 6.0

### DISCUSSION THREAD

**Discussion by pablopaladino** [wagtail/wagtail#11784] [Development]
Hi! I'm trying to upgrade a Wagtail 4.2.4 site to the latest 6.0.1. After all the changes made to the code so there's no conflicts and errors for deprecated functionalities I could run it locally, but when I tried to deploy to my company's development page the GitHub runner threw this error:

```
remote: ModuleNotFoundError: No module named 'wagtail.contrib.modeladmin'

243 remote: ! Error while running '$ python manage.py collectstatic --noinput'.
```

It happened on the robots/wagtail_hooks file. Any ideas how to solve this?

Thanks!

**Comment by gasman** [comment]
See the release note at https://docs.wagtail.org/en/stable/releases/6.0.html#removal-of-modeladmin-app - the modeladmin package is no longer part of Wagtail as of version 6.0, and is available as an external package instead.

**Reply by pablopaladino** [reply]
I changed it to the external package that's wagtail-modeladmin. The problem happens after the runner install the dependencies on the wagtail_hooks file that is being installed like in this error line above the ones I sent prior:

`File "/app/.heroku/python/lib/python3.10/site-packages/robots/wagtail_hooks.py", line 2, in <module>`

whenever I run python manage.py runserver the apps starts with no problems

**Reply by gasman** [reply]
I now see that you also asked this question on https://stackoverflow.com/questions/78208271/problems-deploying-wagtail-6, with some extra details. Please don't ask the same question in multiple places - it means I've spent time filling in information that you already had, which is wasting your time and mine.

**Reply by pablopaladino** [reply]
Ok! thanks for the advice
